### PR TITLE
adding badge -Update README.md

### PR DESCRIPTION
**Pull request by tolgakurtuluss**
Adding codecov badge into "installation" heading part

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ results supported by *easystats*.
 [![see status
 badge](https://easystats.r-universe.dev/badges/see)](https://easystats.r-universe.dev)
 [![R-CMD-check](https://github.com/easystats/see/workflows/R-CMD-check/badge.svg?branch=main)](https://github.com/easystats/see/actions)
+[![codecov](https://codecov.io/gh/easystats/see/branch/master/graph/badge.svg)](https://app.codecov.io/gh/easystats/see)
```
Comment (IndrajeetPatil) [inline README.md:99]:
```suggestion
[![codecov](https://codecov.io/gh/easystats/see/branch/main/graph/badge.svg)](https://app.codecov.io/gh/easystats/see)
```

Comment (IndrajeetPatil) [reply README.md:99]: Actually, this change should be made in the `.Rmd` file and reknit to reflect it in` .md` file

https://github.com/easystats/see/blob/main/README.Rmd

Comment (tolgakurtuluss) [reply README.md:99]: You're right. Now updated my commit on both .Rmd and .md files.

**Comment by IndrajeetPatil** [comment]
Thanks a lot!

**Pull request merged**